? PhysicsTools/PatAlgos/python/cleaningLayer1/__init__.py
? PhysicsTools/PatAlgos/python/mcMatchLayer0/__init__.py
? PhysicsTools/PatAlgos/python/producersHeavyIons/__init__.py
? PhysicsTools/PatAlgos/python/producersLayer1/__init__.py
? PhysicsTools/PatAlgos/python/recoLayer0/__init__.py
? PhysicsTools/PatAlgos/python/selectionLayer1/__init__.py
? PhysicsTools/PatAlgos/python/tools/__init__.py
? PhysicsTools/PatAlgos/python/triggerLayer1/__init__.py
Index: PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc
===================================================================
RCS file: /cvs/CMSSW/CMSSW/PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc,v
retrieving revision 1.44
diff -r1.44 PATElectronProducer.cc
358a359,360
>       anElectron.setEcalDrivenSeed( itElectron->ecalDrivenSeed() );
>       anElectron.setTrackerDrivenSeed( itElectron->trackerDrivenSeed() );
427a430,431
>       anElectron.setEcalDrivenSeed( itElectron->ecalDrivenSeed() );
>       anElectron.setTrackerDrivenSeed( itElectron->trackerDrivenSeed() );
